test/controllers/employment/EmploymentSummaryControllerSpec.scala

```diff
@@ -105,9 +105,9 @@ class EmploymentSummaryControllerSpec extends UnitTestWithApp with MockEmploymen
     )
   }
 
-  lazy val singleView = app.injector.instanceOf[SingleEmploymentSummaryView]
-  lazy val singleEOYView = app.injector.instanceOf[SingleEmploymentSummaryViewEOY]
-  lazy val multipleView = app.injector.instanceOf[MultipleEmploymentsSummaryView]
+  lazy val singleView: SingleEmploymentSummaryView = app.injector.instanceOf[SingleEmploymentSummaryView]
```

Comment:
Those variables should be private.

Reply:
would i be keeping the "lazy"? or just making them private vals?

Reply:
They don't need to be lazy

Reply:
cool, updated to just be private